RiotNSE/BuildSettings.swift

```diff
@@ -242,7 +242,7 @@ final class BuildSettings: NSObject {
     /// Indicates should the app log out the user when number of biometrics failures reaches `maxAllowedNumberOfBiometricsFailures`. Defaults to `false`
     static let logOutUserWhenBiometricsFailuresExceeded: Bool = false
     
-    static let showNotificationsV2: Bool = false
+    static let showNotificationsV2: Bool = true
```

Comment:
I don't understand why this settings is defined for the NSE, I think this is not a pb but this is strange

Reply:
It's defined in Element NSE target too.
And after the next rebase, there will be only one BuildSettings.swift file for all targets.